### ISSUE TITLE
Support IDictionary<string,bool> prefix attributes without having to provide a value

### ISSUE THREAD

**Issue by javiercn** [aspnet/Razor#1589]
Example: (prefix is asp-has-claim)
```html
<tag-helper asp-has-claim-name></tag-helper>
```
Instead of 
```html
<tag-helper asp-has-claim-name="true"></tag-helper>
```

**Comment by rynowak** [comment]
Parking this in the backlog. Eventually we'll groom that and combine all of the collected tag helper suggestions and feedback.

**Comment by rynowak** [comment]
This is a dupe of #431 and was fixed by bd8e9ec
